app/routes/labs.missions/route.tsx

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Grid, GridContainer } from '@trussworks/react-uswds'
+
+import fermiHeroImage from './Fermi_Earth_GWs.jpg'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+import fleet from './AstroFleetChart.png'
+import meatball from '~/components/meatball/logo.svg'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Missions',
+}
+
+export default function () {
+  return (
+    <>
+      <img
+        alt="Fermi satellite with a GRB in the background, both positioned over earth"
+        src={fermiHeroImage}
+      />
+      <GridContainer className="usa-section">
+        <h1>Mutimessenger missions</h1>
+        <h3>Overview</h3>
```

Comment:
It's unusual in writing to have two headings with no paragraph in between.

Reply:
ok, I put a horizontal rule there so there is something between them. 
When looking at the [example website](https://across-web-portal.s3.amazonaws.com/missions.html) they have headings after headings. They used a border-bottom instead of a hr, but using the hr at least keeps the h's from touching.